turbopack/crates/turbopack-core/src/context.rs

```diff
@@ -29,13 +29,14 @@ pub enum ProcessResult {
 #[turbo_tasks::value_impl]
 impl ProcessResult {
     #[turbo_tasks::function]
-    pub fn module(&self) -> Result<Vc<Box<dyn Module>>> {
+    pub async fn module(&self) -> Result<Vc<Box<dyn Module>>> {
         match *self {
             ProcessResult::Module(m) => Ok(*m),
             ProcessResult::Ignore => {
                 bail!("Expected process result to be a module, but it was ignored")
             }
-            ProcessResult::Unknown(_) => {
+            ProcessResult::Unknown(source) => {
+                emit_unknown_module_type_error(*source).await?;
```

Comment:
I think the main reason why I made this bail instead of emit an error, is to have a way to silence the issue for NFT.
https://github.com/vercel/next.js/pull/71713

Reply:
im guessing i don't need it actually and only the strongly_catch_collectables call is needed.  i assumed we were missing an emit since this is where it failed and it failed early,